app/src/main/java/kr/co/connect/boostcamp/livewhere/di/sharedModule.kt

```diff
@@ -0,0 +1,12 @@
+package kr.co.connect.boostcamp.livewhere.di
+
+import kr.co.connect.boostcamp.livewhere.data.SharedPreferenceStorage
+import org.koin.android.ext.koin.androidContext
+import org.koin.dsl.module.module
+
+val sharedModule = module {
```

Comment:
`p3;` 추후 확장용도 목적으로 다움과 같이 이름을 바꾸면 어떨까 싶네요.
```suggestion
val preferencesModule = module {
```